index.html

```diff
@@ -229,14 +229,14 @@ <h2 tabindex="0" class="contact-title">Contact me</h2>
             </div>
             <div class="input-holder">
               <label for="email">Write your email</label>
+              <p class="msg"></p>
```

Comment:
-  Hey, According to the previous review which states below:
:point_down: 
> * [x]  Your Error message comes underneath your input field. Kindly make your error near the submit button like direct below the submit button or directly above the submit button +1 to follow the requirements
> 

- Kindly move the error message before the submit button just after the Textarea field or under the submit button to meet the project requirement.